backend/uclapi/oauth/scoping.py

```diff
@@ -0,0 +1,126 @@
+# Storage of the scope map
+# The purpose of this setup is that the OAuth scope of any app can be stored
+# in a single field. This way, we can easily add more scopes later.
+# We have a BigIntegerField to work with, which means 64 bits of storage.
+# This translates into 64 types of scope, each of which can be checked with a
+# bit mask.
+# E.g. roombookings has scope 0, which is
+# 0000000000000000000000000000000000000000000000000000000000000001b.
+# This is because the 0th bit (LSB) is set to 1.
+# roombookings + uclu = 101b, or a scope number of 2^2 + 2^0 = 4 + 1 = 5
+
+
+class Scopes:
+    SCOPE_MAP = {
+        "roombookings": (0, "Private room bookings data"),
+        "timetable": (1, "Private timetable data"),
+        "uclu": (2, "Private UCLU data"),
+        "moodle": (3, "Private Moodle data")

```

Comment:
I think we should remove all scopes in the scope map that aren't actually in use.

Reply:
that should have been done now. Did I not push it?